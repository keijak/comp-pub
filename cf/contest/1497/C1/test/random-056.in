1
287440516 3
