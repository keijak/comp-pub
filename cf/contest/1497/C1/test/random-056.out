143720256 143720256 4
